cogs/kill.py

```diff
@@ -29,7 +29,7 @@ class ConfirmKillView(View):
     )
     async def confirm_shutdown_button_callback(self, _: discord.Button, interaction: discord.Interaction) -> None:  # noqa: E501
         """When the shutdown button is pressed, delete the message."""
-        logger.debug("\"Confirm\" button pressed. %s", interaction)
+        logger.debug('"Confirm" button pressed. %s', interaction)
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes